docs/src/componentDocs/DrawerFooter/examples/DrawerFooter.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+import {DrawerFooterExample} from './DrawerFooterExample';
+
+const codeSnippet = `<Drawer style={{width: 320, margin: 'auto'}}>
+      <DrawerBody>
+        <DrawerNavItem
+          itemID={'item1'}
+          title={'Account'}
+          icon={{
+            family: 'material-community',
+            name: 'account',
+            direction: 'auto',
+          }}
+        />
+        <DrawerNavItem
+          itemID={'item2'}
+          title={'Notification'}
+          icon={{family: 'material-community', name: 'bell', direction: 'auto'}}
+          activeItemBackgroundShape={'round'}
+        />
+      </DrawerBody>
+      <DrawerFooter>
+        <View style={{padding: 16}}>
+          <View
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              justifyContent: 'space-between',
+              marginBottom: 12,
+            }}>
+            <Text
+              style={{
+                fontSize: 12,
+                color: '#44474E',
+              }}>
+              v2.4.2
+            </Text>
+            <Text style={{fontSize: 12, color: '#44474E'}}>
```

Comment:
use `theme.colors.onSurface` instead of a hardcoded hex value. For reference you can check https://github.com/etn-ccis/blui-react-native/blob/dev/docs/src/componentDocs/ScoreCard/examples/ScoreCardWithBadgeExample.tsx#L13

Reply:
Added as per suggestion